source/github/webhook.go

```diff
@@ -218,10 +218,11 @@ func processPREvent(h *library.Hook, payload *github.PullRequestEvent) (*types.W
 	}
 
 	return &types.Webhook{
-		Comment: "",
-		Hook:    h,
-		Repo:    r,
-		Build:   b,
+		Comment:  "",
+		PRNumber: payload.GetNumber(),
```

Comment:
Now passing through the pull request number with the struct we return to avoid using utility function later to extract that info.

Reply:
Good idea 👍